src/main/java/com/dev/cinema/controller/CinemaHallController.java

```diff
@@ -0,0 +1,37 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.dto.CinemaHallRequestDto;
+import com.dev.cinema.service.CinemaHallService;
+import com.dev.cinema.service.impl.mapper.CinemaHallMapper;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallMapper cinemaHallMapper;
+
+    @Autowired
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallMapper cinemaHallMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallMapper = cinemaHallMapper;
+    }
+
+    @PostMapping
+    public void create(@RequestBody CinemaHallRequestDto cinemaHallRequestDto) {
+        cinemaHallService.add(cinemaHallMapper.convert(cinemaHallRequestDto));
+    }
+
+    @GetMapping
+    public List<CinemaHall> getAll() {
+        return cinemaHallService.getAll();
```

Comment:
You must use CinemaHallResponseDto instead of CinemaHall

Reply:
done